File: net/src/Sdl.Web.GraphQL/Response/GraphQLError.cs
using System.Collections.Generic;

namespace Sdl.Web.GraphQLClient.Response
{
    public class GraphQLError
    {
        public string Message { get; set; }
        public List<object> Path { get; set; }
        public List<GraphQLErrorLocation> Locations { get; set; }
        public GraphQLExtensions Extensions { get; set; }
    }
}

File: java/src/main/java/com/sdl/web/pca/client/response1/GraphQLError.java
package com.sdl.web.pca.client.response;

import java.util.List;

public class GraphQLError {
    public String Message;
    public List<Object> Path;
    public List<GraphQLErrorLocation> Locations;
    public GraphQLExtensions Extensions;

    public void setMessage(String message) {
        Message = message;
    }

    public String getMessage() {
        return Message;
    }

    public void setPath(List<Object> path) {
        Path = path;
    }

    public List<Object> getPath() {
        return Path;
    }

    public void setLocations(List<GraphQLErrorLocation> locations) {
        Locations = locations;
    }

    public List<GraphQLErrorLocation> getLocations() {
        return Locations;
    }

    public void setExtensions(GraphQLExtensions extensions) {
        Extensions = extensions;
    }

    public GraphQLExtensions getExtensions() {
        return Extensions;
    }
}
